frontend/packages/dev-console/console-extensions.json

```diff
@@ -1329,5 +1328,16 @@
       "path": ["/search-page/all-namespaces", "/search-page/ns/:ns"],
       "component": { "$codeRef": "common.SearchPage" }
     }
+  },
+  {
+    "type": "console.navigation/section",
+    "properties": {
+      "id": "ecosystem",
+      "perspective": "admin",
+      "section": "ecosystem",
+      "name": "%devconsole~Ecosystem%",
+      "href": "/ecosystem",
+      "insertAfter": "operators"
```

Comment:
`operators` is eventually going away, so it would be better to put this after `Favorites`.